lib/blocks.php

```diff
@@ -220,7 +220,13 @@ function gutenberg_wpautop_block_content( $content ) {
  */
 function gutenberg_wpautop_insert_post_data( $data ) {
 	if ( ! empty( $data['post_content'] ) && gutenberg_content_has_blocks( $data['post_content'] ) ) {
+		// WP_REST_Posts_Controller slashes post data before inserting/updating
+		// a post. This data gets unslashed by `wp_insert_post` right before
+		// saving to the DB. The PEG parser needs unslashed input in order to
+		// properly parse JSON attributes.
+		$data['post_content'] = wp_unslash( $data['post_content'] );
 		$data['post_content'] = gutenberg_wpautop_block_content( $data['post_content'] );
+		$data['post_content'] = wp_slash( $data['post_content'] );
```

Comment:
Minor but It might be better to avoid storing any unslashed data in `$data[ 'post_content' ]` and use a temporary variable.